crates/evm/evm/src/executors/invariant/error.rs

```diff
@@ -296,153 +264,39 @@ impl FailedInvariantCaseData {
         // 2. Bob calls transferOwnership to Alice
         // 3. Alice calls acceptOwnership and test fails
         // we shrink to indices of [2, 1] and we recreate call sequence in same order.
-        Ok(shrunk_call_indices.iter().map(|idx| &calls[*idx]).collect())
+        Ok(shrinker.current().iter().map(|idx| &calls[*idx]).collect())
```

Comment:
we make a vec of references here, and then above we clone and make a vec of owned txs; they should just be cloned here or return an iterator instead of collecting

Reply:
changed in https://github.com/foundry-rs/foundry/pull/7756/commits/55a44ba8ebb57adb65086789cbf71d3f1f79cd1d